src/Map.jsx

```diff
@@ -75,7 +75,14 @@ class Map extends Component {
       { force: true },
     );
   }
-
+  getStopsWithSid(route) {
+    const stops = this.state.selectedStops;
+    const indexArray = [];
+    indexArray.push(route.stops.findIndex(stop => stop.sid === stops[0].sid));
+    indexArray.push(route.stops.findIndex(stop => stop.sid === stops[1].sid));
+    indexArray.sort((a, b) => a - b);
+    return route.stops.filter((stop, index) => index >= indexArray[0] && index <= indexArray[1]);
```

Comment:
Will make this more elegant; can be done in less lines

Reply:
++